src/test/regress/expected/subquery_view.out

```diff
@@ -584,16 +584,11 @@ EXPLAIN (COSTS OFF) WITH cte AS (
   FROM pg_stat_activity
 ) SELECT * FROM reference_table JOIN cte USING (text_col);
```

Comment:
Hmm, these were added as part of a bugfix (or to have more coverage for a bug fix): https://github.com/citusdata/citus/pull/4418

Basically, it was problematic to pushdown queries that involve joins of pg_catalog tables and Citus reference/dist tables.

It seems like when the coordinator is added to the metadata, we locally execute such queries. I wonder if that is a problem.

Can you open an issue to track this as well? I think this might require further discussion when we decide to require all clusters to have the coordinator in the metadata.

The issue you open could be as simple as sharing these queries whose output changed, and show that the behavior/explain output is diffierent when the coordinator is in the metadata.

Reply:
Taking a closer look at this, when coordinator is in metadata all the reference tables are replicated to the coordinator as well. So for the queries in question such as the one below, all the data is available on coordinator locally. Running those queries locally looks like something expected to me.  

```
CREATE OR REPLACE VIEW view_on_views AS SELECT pg_stat_activity.application_name, pg_locks.pid FROM pg_stat_activity, pg_locks;

WITH cte AS (
  SELECT application_name AS text_col
  FROM view_on_views
) SELECT * FROM reference_table JOIN cte USING (text_col);
```
But we may want to add tests to cover subquery pushdown since we are not covering it here anymore. What do you think?